cime/config/e3sm/allactive/config_pesall.xml

```diff
@@ -1,5 +1,232 @@
 <?xml version="1.0"?>
 <config_pes>
+  <!-- ===================================================================== -->
```

Comment:
Just a note that we determined through Slack conversation that this chunk of additions needs to be moved to the end.